File: java/blackjack/Card.java
package blackjack;

/**
 * Created by 1mike12 on 12/16/2016.
 */
public class Card {

    static String description = "Glossy paper card with blue backing";

    String suite;
    int rank;

    Card(String suite, int rank) {
        this.suite = suite;
        this.rank = rank;
    }

    public String toString() {
        return this.suite + String.valueOf(this.rank);
    }

    int getScore() {

        if (rank == 1) {
            return 11;
        } else if (rank >= 10) {
            return 10;
        } else {
            return rank;
        }
    }
}

File: python2/blackjack/Card.py
class Card(object):
    # suite C, H, S, D

    # rank 1-13


    def __init__(self, suite, rank):
        self.suite = suite
        self.rank = rank

        score = -1

        if rank == 1:
            score = 11 or 1
        elif rank >= 10:
            score = 10
        else:
            score = rank

        self.score = score

    def __repr__(self):
        return self.suite + " " + str(self.rank) + " " + str(self.score)
